src/components/Form/Form.js

```diff
@@ -83,7 +109,25 @@ const StyledTextarea = styled.textarea`
 	border-radius: 5px;
 	color: var(--turq);
 `;
-
 const StyledLabel = styled.label`
 	display: none;
 `;
+const Styledh2 = styled.h2`
+	color: var(--salmon);
+	font-size: ${pxToRem(30)};
+`;
+const StyledDiv = styled.div`
```

Comment:
I don't really think that calling it "StyledDiv" is a good idea. As your app grows you will most likely have more than one component that's a styled div. The name will be confusing than. As always, I'd suggest a more specific name. Maybe RadioButtonBox or RadioButtonDiv?

Reply:
@ckskr or in case you want to keep the div in the name, make it **_MoodButtonBoxDiv_**.